Makefile

```diff
@@ -488,7 +514,10 @@ $(VERSRC): Makefile
 # 		- run `make tag` and use its output to 'git tag' and 'git push --tags'
 .PHONY: tag
 tag:
-	@echo "Run this:\n> git tag $(GITTAG)\n> git push --tags"
+	@echo 'Run this: \
+	> git tag $(GITTAG) && \
+	  git tag api/$(GITTAG) && \
+	  git push --tags'
```

Comment:
You want to do `git push origin $(GITTAG) && git push origin api/$(GITTAG)`.

The current command pushes all tags you may have locally to `origin` which is not what you want.